onnxruntime/core/providers/tensorrt/tensorrt_execution_provider.cc

```diff
@@ -2787,15 +2805,18 @@ Status TensorrtExecutionProvider::CreateNodeComputeInfoFromGraph(const GraphView
   if (fp16_enable_ && int8_enable_) {
     trt_config->setFlags(1U << static_cast<uint32_t>(nvinfer1::BuilderFlag::kFP16) | 1U << static_cast<uint32_t>(nvinfer1::BuilderFlag::kINT8));
     trt_node_name_with_precision += "_fp16_int8";
-    LOGS_DEFAULT(VERBOSE) << "[TensorRT EP] FP16 and INT8 mode is enabled";
+    strong_typing_enable_ = false;LOGS_DEFAULT(VERBOSE) << "[TensorRT EP] FP16 and INT8 mode is enabled";
   } else if (fp16_enable_) {
     trt_config->setFlag(nvinfer1::BuilderFlag::kFP16);
     trt_node_name_with_precision += "_fp16";
-    LOGS_DEFAULT(VERBOSE) << "[TensorRT EP] FP16 mode is enabled";
+    strong_typing_enable_ = false;LOGS_DEFAULT(VERBOSE) << "[TensorRT EP] FP16 mode is enabled";
   } else if (int8_enable_) {
     trt_config->setFlag(nvinfer1::BuilderFlag::kINT8);
-    trt_node_name_with_precision += "_int8";
+    trt_node_name_with_precision += "_int8";strong_typing_enable_ = false;
     LOGS_DEFAULT(VERBOSE) << "[TensorRT EP] INT8 mode is enabled";
+  } else {
+      strong_typing_enable_ = true;
```

Comment:
Not sure the effect of setting `strong_typing_enable_ = true` since TRT EP won't call trt_builder->createNetworkV2 afterwards